styleguide.config.js

```diff
@@ -42,6 +42,10 @@ module.exports = {
           content:
             './packages/design-system/src/components/NotificationSystem/NotificationSystem.md'
         },
+        {
+          name: 'Badge',
+          components: './src/components/Badge/Badge.js'
```

Comment:
the url is wrong and the component is not visible in the docs
it should be: `./packages/design-system/src/components/Badge/Badge.js`